JWT_Auth/src/utils/helper.js

```diff
@@ -0,0 +1,40 @@
+import * as fs from "fs";
+import * as fsPromise from "fs/promises";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const dbFilePath = process.env.DB_PATH;
+
+export const handleResponseSend = async (res, message, code) => {
+  res.statusCode = code;
+  return res.end(JSON.stringify({ message }));
+};
+
+export const getFileData = async (res) => {
+  try {
+    // check file exists or not
+    if (!fs.existsSync(dbFilePath)) {
+      // create a file for user data
+      await fsPromise.writeFile(dbFilePath, "[]", { encoding: "utf-8" });
+    }
+
+    // return users data from file.
+    let fileData = await fsPromise.readFile(dbFilePath, {
```

Comment:
this is correct but you can also read file directly by importing

Reply:
ok